commit ffac09556f6f3d5d758bac9ba2b91117e6014ea7
Author: Christoph Feck <christoph@maxiom.de>
Date:   Fri Nov 2 17:31:48 2012 +0100

    SVN_SILENT link
    
    If possible, please include this build fix in 4.9.3 tag.
    
    BUG: 308939
    CCMAIL: kde-release-team@kde.org

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 03eabf3..c8f425a 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -138,7 +138,7 @@ target_link_libraries(kalzium ${KDE4_KHTML_LIBRARY} ${KDE4_KNEWSTUFF3_LIBS} ${KD
 if (OPENBABEL2_FOUND)
    target_link_libraries(kalzium ${OPENBABEL2_LIBRARIES})
    if (EIGEN2_FOUND AND Avogadro_FOUND)
-      target_link_libraries(kalzium compoundviewer avogadro)
+      target_link_libraries(kalzium ${QT_QTOPENGL_LIBRARY} compoundviewer avogadro)
    endif (EIGEN2_FOUND AND Avogadro_FOUND)
 endif (OPENBABEL2_FOUND)
 
